index.js

```diff
@@ -128,8 +127,8 @@ module.exports = function (content) {
     generatorOptions.cssTemplate = path.resolve(this.context, fontConfig.cssTemplate);
   }
 
-  if (fontConfig.cssFontsPath) {
-    generatorOptions.cssFontsPath = path.resolve(this.context, fontConfig.cssFontsPath);
+  if (fontConfig.cssFontsUrl) {
+    generatorOptions.cssFontsUrl = path.resolve(this.context, fontConfig.cssFontsUrl);
```

Comment:
I see you update `webfonts-generator` but the new one still uses `cssFontsPath` ([see here](https://github.com/vusion/webfonts-generator#cssfontspath)), right? I'm afraid here `generatorOptions.cssFontsUrl` will be ignored by the generator

Reply:
Please look https://github.com/vusion/webfonts-generator/blob/master/src/index.js#L45